kubernetes/kserve/tests/scripts/test_mnist.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+echo "MNIST KServe V2 test begin"
+
+echo "Removing any previous kubernetes cluster "
+minikube delete
+
+echo "Starting kubernetes cluster "
+
+minikube start
+
+echo "Install kserve"
+
+cd $GITHUB_WORKSPACE/kserve
+./hack/quick_install.sh
+echo "Waiting 5s for kserve pod to come up ..."
+sleep 5
+
+echo "Deploy the cluster"
+
+cd $GITHUB_WORKSPACE
+kubectl apply -f kubernetes/kserve/tests/configs/mnist_v2_cpu.yaml
+
+echo "Waiting 120s for pod to come up..."
+sleep 120
```

Comment:
I don't like these sleep staetments, can you poll the pod status with kubectl or minikube instead?

Reply:
Done. Refactored the code to be modular